modules/engine/src/model/model.ts

```diff
@@ -227,7 +237,8 @@ export class Model {
 
   // Draw call
 
-  draw(renderPass: RenderPass): void {
+  // TODO(v9): Should draw() take 'transformFeedback' in options?
+  draw(renderPass: RenderPass, options?: {transformFeedback?: TransformFeedback}): void {
```

Comment:
Open question here -- do we want a different API at this stage?

Reply:
We should probably handle this the same way we do other resources. I.e. let the application set a transform feedback on the model with props, setProps, or a dedicated method before the draw call.  See how vertexArray is handled.